docs/translations/es/what-is-gaia.md

```diff
@@ -0,0 +1,26 @@
+<!--
+order: 1
+-->
+
+# ¿Qué es Gaia?
+
+`gaia` es el nombre de la aplicación de Cosmos SDK para el Hub de Cosmos. Viene con 2 opciones principales de entrada:
+
+- `gaiad`: El demonio de Gaia, ejecuta un full-node con la aplicacion `gaia`.
+- `gaiacli`: Interface de línea de comandos de Gaia, la cual activa la interacción con en full-node de Gaia.
+
+`gaia` es construida con el SDK de Cosmos usando los siguientes módulos:
+
+- `x/auth`: Cuentas y firmas.
```

Comment:
esta lista está desactualizada. Podrías añadir los nuevos modulos que están en `master`?

Reply:
Disculpa, he estado buscando pero no se muy bien a qué lista actualizada te refieres, he buscado en la [documentación](https://github.com/cosmos/gaia/blob/master/docs/gaia-tutorials/what-is-gaia.md), en `/gaia/cmd/gaiacli/` [main.go](https://github.com/cosmos/gaia/blob/master/cmd/gaiacli/main.go) y en `gaia/app/` [app.go](https://github.com/cosmos/gaia/blob/master/app/app.go)
Si me comentas cual estaría encantado de actualizarla, gracias!

Reply:
Gaia ahora utiliza más módulos de la SDK. No es necesario que escribas una descripción pero sí que estén en la lista:

https://github.com/cosmos/cosmos-sdk/tree/master/x

Reply:
ok modificado, ves bien añadir una pequeña descripción así:

- `x/capability`: Permite el aprovisionamiento, el seguimiento y la autentificación de las capacidades de múltiples propietarios en el tiempo de ejecución.

- `x/crisis`: _Can't find the explanation on [module](https://github.com/cosmos/cosmos-sdk/tree/master/x/crisis)_.

- `x/evidence`: Permite la presentación y manipulación de pruebas arbitrarias de mal comportamiento.

- `x/genaccounts`: Está obsoleto, este módulo ha sido reemplazado por [ADR 011: Generalize Module Accounts](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-011-generalize-genesis-accounts.md)

- `x/mint`: _Can't find the explanation on [module](https://github.com/cosmos/cosmos-sdk/tree/master/x/mint)_.

- `x/simulation`: Implementa una aplicación completa con el SDK de Cosmos utilizada para ejecutar pruebas de simulación.

- `x/upgrade`: Proporciona un módulo para el SDK de Cosmos que puede ser usado para actualizar sin problemas una cadena activa de Cosmos a una nueva versión de software.